content/english/supply chain analytics/2_Scenario.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Setting the Scene"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 3
+---
+
+The new Xbox 1000 has just come out, and you NEED to get it. But to do so you’ll need to start saving a lot more than the money. After all the kids in school loved the cookies your mom baked for the school fundraiser, you had the genius idea to start up a bakery with her and split the profits 50-50!  
+
+To start off, you’re just going to sell 2 cookies. After polling your friends, the top two cookies everyone loves are chocolate chip and sugar cookies.
+
+Each batch of cookies is 12 cookies. You can sell one batch of chocolate chip cookies for $20 and one batch of sugar cookies for $15.
+
+
+![Alt text](img/cookie%20offerings.jpg)
+
+
+You need to buy the ingredients to make the cookies. In supply chain, these are known as “raw materials”. When you did the math, you found that since chocolate chip cookies use more ingredients than sugar cookies, each batch will cost you $10 in raw materials. Each batch of sugar cookies will only cost $7. 
+
+The cookie business is very competitive. To stay in business, you need to make a profit, which means you make more money than you spend. 
+
+{{% notice tip %}}
+
+How do you calculate profit?
+## Profit = Revenue - Cost
+
+{{% /notice %}}
+
+Here’s the catch- You only have a limited number of common ingredients that you have to share between both of the cookies. But each batch requires a different amount. To make it simple, I’ve only included 3 of the main ingredients: eggs, flour, and sugar. 
+
+![Alt text](img/constrained%20ingredients.jpg)
```

Comment:
We could also use a markdown table if you like - this makes it a bit easier for visually impaired folks